### ISSUE TITLE
not clear what rescueCells_temp() function is

### ISSUE THREAD

**Issue by romunov** [dpcook/emt_dynamics#2]
A function named `recueCells_temp()` is used in several places (i.e. [here](https://github.com/dpcook/emt_dynamics/blob/master/processing_notebooks/timecourse_processing.Rmd#L526)) but not defined. Tried searching for it using the search functionality and manually go through all the notebooks to no avail. It's not  clear how this function differs from the `deMULTIplex::rescueCells`.

**Comment by dpcook** [comment]
Hey @romunov -- really sorry about this. There should be no issue using the standard  `deMULTIplex::rescueCells` in place here. I had done this processing in the early days of MULTI-seq and `rescueCells` had yet to be implemented in their `deMULTIplex` package, but Chris sent me some sample source code for the function to try out. 

Although I can't recall for certain, I suspect I just named the function awkwardly because I was just trialing it out. No steps of the actual function were modified and [in subsequent MULTI-seq scripts](https://github.com/dpcook/thebaud_lung_BPD/blob/a82fcf5bc9996a6ed9567f605c9252ba9693b55c/processing.Rmd#L392), I've simply replaced this with the standard `rescueCells`.

Sorry for the confusion!
